src/main/java/net/neoforged/neoforge/event/entity/item/ItemTossEvent.java

```diff
@@ -21,11 +21,11 @@ public class ItemTossEvent extends ItemEvent implements ICancellableEvent {
     /**
      * Creates a new event for EntityItems tossed by a player.
      * 
-     * @param entityItem The EntityItem being tossed.
+     * @param itemEntity The EntityItem being tossed.
```

Comment:
Don't forget this EntityItem here too and above. We call them ItemEntity as that is what the actual class is. Someone must had typo'ed EntityItem before